src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/_exception_handler.py

```diff
@@ -39,15 +39,15 @@ def invalid_arg_found_exception_handler(ex):
                     msg = msg['message'].split('\n')[0]
                     msg = msg[len('Message: '):] if msg.find('Message: ') == 0 else msg
                     cli_error = CLIError('Operation Failed: Invalid Arg '
-                                         '(Server returned status code 400 {})'.format(str(msg)))
+                                         '(Server returned status code 400) {}'.format(msg))
```

Comment:
_Non-blocking opinion_ As an error message intended for a user, HTTP response status code is not a clear sign of what caused the failure. Aside from it is a client side error, it offers little information. I do not expect all users to be familiar with Azure Restful API. Instead, the customer would like to know what cause this issue and how to recover from it. In that case, the error message should have a more prominent place in the exception message body. I would suggest sending the status code to logger as an error level message.